src/ophyd_async/core/sim_signal_backend.py

```diff
@@ -165,6 +165,9 @@ async def get_reading(self) -> Reading:
     async def get_value(self) -> T:
         return self._get_value()
 
+    async def get_setpoint(self) -> T:
+        return self.converter.value(self._value)
```

Comment:
is it worth making this just call get_value, and adding a comment that for a simsignalbackend these will always be the same?

i.e.:

```suggestion
    async def get_setpoint(self) -> T:
        """For a simulated backend, the setpoint and readback values are the same."""
        return await self.get_value()
```